charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-snapshot-validation-webhook-networkpolicy.yaml

```diff
@@ -1,26 +0,0 @@
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  labels:
-    app: snapshot-validation
-    gardener.cloud/role: controlplane
-  name: allow-kube-apiserver-to-csi-snapshot-validation
-  namespace: {{ .Release.Namespace }}
-  annotations:
-    gardener.cloud/description: "Allows Egress from shoot's kube-apiserver pods to csi-snapshot-validation pods."
-spec:
-  egress:
-  - ports:
-    - port: 443
-      protocol: TCP
-    to:
```

Comment:
without this network policy , can shoot api server pod call csi snapshot validation webhook in shoot control plane namespace?

Reply:
This is ensured by https://github.com/gardener/gardener-extension-provider-alicloud/pull/581/commits/3f1780cc1f51c57f83007de521839bb003b4b296#diff-cc0552e76e65ffff14fa4d53e143f0d325fb62ff3c926693a9153658dcbfd23dR55 now.